specification/eventgrid/data-plane/Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json

```diff
@@ -400,6 +404,29 @@
         }
       }
     },
+    "AcsRecordingFileStatusUpdatedEventData": {
+      "description": "Schema of the Data property of an EventGridEvent for an Microsoft.Communication.RecordingFileStatusUpdated event.",
+      "properties": {
+        "recordingStorageInfo": {
+          "description": "The details of recording storage information",
```

Comment:
why double indirection? recordingStorageInfo.recordingChunks? Do we intend to add more properties in to AcsRecordingStorageInfoProperties?

Reply:
Not right now but that is the schema we got from Call Recording team